c++/src/capnp/rpc.c++

```diff
@@ -1190,6 +1200,11 @@ private:
 
     VoidPromiseAndPipeline call(uint64_t interfaceId, uint16_t methodId,
                                 kj::Own<CallContextHook>&& context, CallHints hints) override {
+      if (isResolved()) {
+        // Same as in newCall().
+        return cap->call(interfaceId, methodId, kj::mv(context), hints);
+      }
+
       receivedCall = true;
       return cap->call(interfaceId, methodId, kj::mv(context), hints);
```

Comment:
Not sure how this is different? Just avoiding the `receivedCall` field setting.

Reply:
Yeah you're right, it has no actual effect here, I was just trying to be consistent I guess.

Reply:
(The setting of `receivedCall` doesn't matter post-resolve, since it only impacts what happens at resolve time.)